lua/mini/misc.lua

```diff
@@ -409,6 +409,137 @@ MiniMisc.zoom = function(buf_id, config)
   end
 end
 
+--- Restore the cursor on file reopen
+---
+--- What it does:
+--- - When reopening a file, this will make sure the cursor is placed back to
+---   the position where you left before.
+--- - This implements :help restore-cursor in a nicer way.
+--- - Creates autocommand for |BufReadPre| and |FileType| event with
+---   |MiniMisc.restore_cursor|
```

Comment:
Merge, please, two first points in a single one. No need for implementation details about autocommands.

Reply:
Done.